moonshot/src/redteaming/session/session.py

```diff
@@ -298,3 +363,77 @@ async def run(self) -> dict:
         # ------------------------------------------------------------------------------
         print("[Session] Part 4: Wrap up run...")
         return runner_results
+
+    def check_redteaming_type(self) -> str:
```

Comment:
doc string, and can the comparison be an enum like yyy == RedTeamingType.MANUAL.name

Reply:
updated to return RedTeamingType enum name